Modules/socketmodule.c

```diff
@@ -8187,6 +8187,48 @@ socket_exec(PyObject *m)
 
     ADD_INT_MACRO(m, J1939_FILTER_MAX);
 #endif
+#ifdef HAVE_LINUX_CAN_ISOTP_H
+    PyModule_AddIntMacro(m, SOL_CAN_ISOTP);
```

Comment:
Please use the `ADD_INT_MACRO` wrapper as above; now, you're discarding the return value, thus possibly leaking resources, for every `PyModule_AddIntMacro` invocation.